components/platiagro-notebook-servers/experiment/docker-entrypoint.py

```diff
@@ -265,10 +259,6 @@ def main():
     if dataset:
         save_dataset(dataset)
 
-    if exception is not None:
-        logging.error("Exception raised when running a notebook.")
-        raise exception
```

Comment:
@dnlcesilva tira só o `logging.error`, por favor. O `if exception is not None` e o `raise exception` ainda são necessários.

Reply:
@fberanizo ok!